bedrock/mozorg/views.py

```diff
@@ -234,7 +234,40 @@ class ContributeIndex(l10n_utils.LangFilesMixin, TemplateView):
     template_name = 'mozorg/contribute/index.html'
 
 
+class ContributeSignupNew(TemplateView):
+
+    def get_template_names(self):
+        variant = self.request.GET.get('v', '')
+        template = 'mozorg/contribute/signup-new.html'
+
+        if variant == '2':
+            template = 'mozorg/contribute/signup-alternate.html'
+
+        return [template]
+
+
+class ContributeTaskView(TemplateView):
+
+    def get_template_names(self):
+        variant = self.request.GET.get('v', '')
+        task = filter(None, self.request.path.split('/'))[-1]
+
+        # if the variant was not set, this is a v1 variant
+        if not variant:
+            variant = '1'
+
+        template = 'mozorg/contribute/tasks/v{0}/{1}.html'.format(variant, task)
```

Comment:
For safety's sake, should make sure both `variant` and `task` contain valid/expected values. Something like:

```python
class ContributeTaskView(TemplateView):
    tasks = [
        'build-firefox',
        'devtools-challenger',
        'firefox-mobile',
        'follow-mozilla',
        'joy-of-coding',
        'whimsy',
    ]

    def get_template_names(self):
        variant = self.request.GET.get('v', '1')
        task = filter(None, self.request.path.split('/'))[-1]

        if variant in ['1', '2'] and task in self.tasks:
            template = 'mozorg/contribute/tasks/v{0}/{1}.html'.format(variant, task)
        else:
            raise Http404

        return [template]
```